### PR TITLE
Add SignalR background service 6.0 sample

### LINKED ISSUE #25155: .NET 6:  Host ASP.NET Core SignalR in background services
[This page ](https://docs.microsoft.com/en-us/aspnet/core/signalr/background-services?view=aspnetcore-6.0#enable-signalr-in-startup), has not been updated for ASPNetCore 6.

**EDIT by @Rick-Anderson**

Consider adding SLN file. [.gitignore](https://github.com/dotnet/AspNetCore.Docs/blob/main/.gitignore#L15-L17) prevents `.sln` files from being checked in, so follow the instructions to force check in of `.sln` file.

```
# Use git add -f to force override .sln when required. Not needed in most cases.
# git add -f myProj.sln
```

Add readme.md file explaining what each project is for, how to test, etc.

See https://github.com/dotnet/AspNetCore.Docs/pull/25326#issuecomment-1069004348

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1bbb2d1c-4dfe-3faf-d4b0-65ae1146961e
* Version Independent ID: 6b203f18-adfa-3a18-183d-b124325064c3
* Content: [Host ASP.NET Core SignalR in background services](https://docs.microsoft.com/en-us/aspnet/core/signalr/background-services?view=aspnetcore-6.0)
* Content Source: [aspnetcore/signalr/background-services.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/signalr/background-services.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-signalr**
* GitHub Login: @bradygaster
* Microsoft Alias: **bradyg**

### PR DESCRIPTION
**Pull request by UncleDave**
Fixes #25155.

[Internal Review](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/background-services?view=aspnetcore-6.0&branch=pr-en-us-25361).

### CODE REVIEW

File: aspnetcore/signalr/background-service/samples/6.0/readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+* 2 console applications to consume the output from the web application via SignalR.
+
+## Running this sample
+Start the `Server`, `Clients.ConsoleOne`, and `Clients.ConsoleTwo` projects with your favourite IDE, or run the following commands from this folder in separate terminals:
+
+* `dotnet run --project Server`
+* `dotnet run --project Clients.ConsoleOne`
+* `dotnet run --project Clients.ConsoleTwo`
+
+The `ConsoleOne` and `ConsoleTwo` applications will produce log output when they receive a message via SignalR.
```
Comment (Rick-Anderson) [inline aspnetcore/signalr/background-service/samples/6.0/readme.md:16]:
```suggestion
The `ConsoleOne` and `ConsoleTwo` applications produce log output when they receive a message via SignalR.
```

File: aspnetcore/signalr/background-services.md [hunk 2]
```diff
@@ -20,47 +20,47 @@ This article provides guidance for:
...
+Hosting ASP.NET Core SignalR Hubs in the context of a background worker process is identical to hosting a Hub in an ASP.NET Core web app. In `Program.cs`, calling `builder.Services.AddSignalR` adds the required services to the ASP.NET Core Dependency Injection (DI) layer to support SignalR. The `MapHub` method is called on the `WebApplication` `app` to connect the Hub endpoints in the ASP.NET Core request pipeline.
 
-[!code-csharp[Startup](background-service/samples/3.x/Server/Startup.cs?name=Startup)]
+[!code-csharp[Program](background-service/samples/6.0/Server/Program.cs?name=Program)]
 
-In the preceding example, the `ClockHub` class implements the `Hub<T>` class to create a strongly typed Hub. The `ClockHub` has been configured in the `Startup` class to respond to requests at the endpoint `/hubs/clock`.
+In the preceding example, the `ClockHub` class implements the `Hub<T>` class to create a strongly typed Hub. The `ClockHub` has been configured in `Program` to respond to requests at the endpoint `/hubs/clock`.
```
Comment (Rick-Anderson) [inline aspnetcore/signalr/background-services.md:32]:
```suggestion
In the preceding example, the `ClockHub` class implements the `Hub<T>` class to create a strongly typed Hub. The `ClockHub` has been configured in `Program.cs` to respond to requests at the endpoint `/hubs/clock`.
```

**Review by Rick-Anderson — APPROVED** [review]
Nice work, really helpful

File: aspnetcore/signalr/background-services.md [hunk 3]
```diff
@@ -20,47 +20,47 @@ This article provides guidance for:
 
 :::moniker range=">= aspnetcore-6.0"
 
-[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/signalr/background-service/samples/3.x) [(how to download)](xref:index#how-to-download-a-sample)
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/signalr/background-service/samples/6.0) [(how to download)](xref:index#how-to-download-a-sample)
 
 ## Enable SignalR in startup
```
Comment (Rick-Anderson) [inline aspnetcore/signalr/background-services.md:25]:
```suggestion
## Enable SignalR at app startup
```

**Review by Rick-Anderson — APPROVED** [review]

**Review by serpent5 — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
Fantastic work @UncleDave 
Do you have a twitter or other account you'd like to use for your author link?
Would you be willing to help on other SignalR updates?

**Comment by UncleDave** [comment]
> Fantastic work @UncleDave Do you have a twitter or other account you'd like to use for your author link? Would you be willing to help on other SignalR updates?

Thanks. I don't use twitter or anything like that so GitHub is fine, though I don't have much on here either 😅.
I'm by no means an expert on SignalR but I could certainly look if you were to throw something my way.

**Pull request merged**